.github/workflows/ci.yml

```diff
@@ -3,10 +3,10 @@ name: CI
 on:
   pull_request:
     branches:
-      - master
+      - teleport
```

Comment:
Why are we switching the branch here?

Reply:
Because `teleport` is our actual branch. For forks, we create a separate `teleport` branch. Without this change, CI doesn't run.